### PR TITLE
Add molecule for role icinga_plugins

### PR DESCRIPTION
**Pull request by beechesII**
* added molecule/default/converge.yml and molecule/default/molecule.yml to role icinga_plugins
* changed molecule/default/converge.yml
* added molecule test for role icinga_plugins to CI.yml
* changed distro from Redhat to RedHat 

Updates #17

### CODE REVIEW

File: roles/icinga_plugins/molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+- name: Converge
+  hosts: all
+
+  pre_tasks:
```
Comment (rndmh3ro) [inline roles/icinga_plugins/molecule/default/converge.yml:5]: Could these go into the `prepare` stage? The prepare stage only gets executed once and does not have to be idempotent.

Comment (rndmh3ro) [reply roles/icinga_plugins/molecule/default/converge.yml:5]: Also isn't there a role to do all this?

File: roles/icinga_plugins/molecule/default/converge.yml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          - monitoring-plugins
+      when: ansible_os_family == 'Debian'
+
+  collections:
+    - t_systems_mms.ansible_collection_icinga
+
+  roles:
+    - icinga_plugins
+
+  post_tasks:
```
Comment (rndmh3ro) [inline roles/icinga_plugins/molecule/default/converge.yml:103]: These tasks look like they are tests. If it is so, they belong into the `verify`-phase.

File: roles/icinga_plugins/molecule/default/converge.yml (hunk 1 above)
Comment (beechesII) [reply roles/icinga_plugins/molecule/default/converge.yml:6]: > Could these go into the `prepare` stage? The prepare stage only gets executed once and does not have to be idempotent.

done

Comment (beechesII) [reply roles/icinga_plugins/molecule/default/converge.yml:6]: > Also isn't there a role to do all this?

Normally icinga_agent role will install the icinga2 package, but we dont want to mix the tests for these roles.

**Pull request merged**